ui/components/DashboardComponent.js

```diff
@@ -797,48 +808,51 @@ class DashboardComponent extends React.Component {
 
       showAdapters = (
         <div>
-          {availableAdapters.map((aa, ia) => {
-            let isDisabled = true;
-            let image = "/static/img/meshery-logo.png";
-            let logoIcon = <img src={image} className={classes.icon} />;
-            let adapterType = "";
-            let adapterVersion = "";
-            meshAdapters.forEach((adapter) => {
-              if (aa.value === adapter.adapter_location) {
-                isDisabled = false;
-                adapterType = adapter.name;
-                adapterVersion = adapter.version;
-                image = "/static/img/" + adapter.name.toLowerCase() + ".svg";
-                logoIcon = <img src={image} className={classes.icon} />;
-              }
-            });
+          {
+            meshAdapters.map((runningMeshAdapter, idx) => {
+              const { label, value } = availableAdapters.find(adapter => adapter.value === runningMeshAdapter.adapter_location)
+              const adapterName = capitalizeString(runningMeshAdapter.name)
+              const logo = "/static/img/" + runningMeshAdapter.name.toLowerCase() + ".svg";
+
+              return (
+                <Tooltip
+                  key={idx}
+                  title={`${adapterName} adapter version ${runningMeshAdapter.version} on port ${label.split(":")[1]}`}
+                >
+                  <Chip
+                    label={adapterName}
```

Comment:
Note to reviewer:
using Adaper name from available mesh adapter informations, unlike earlier where adapter location was used at the same place. Let me know if we don't need it.